engine/include/engine/graphics/PostProcessingHandler.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef POSTPROCESSINGHANDLER_HPP
+#define POSTPROCESSINGHANDLER_HPP
+
+#include <engine/resources/Shader.hpp>
+
+class PostProcessingHandler {
+private:
+    PostProcessingHandler() = default;
+    ~PostProcessingHandler() {}
+    PostProcessingHandler(const PostProcessingHandler&) = delete;
+    PostProcessingHandler& operator=(const PostProcessingHandler&) = delete;
+
+    static unsigned int hdrFBO;
+    static unsigned int colorBuffers[2];
+    static unsigned int attachments[2];
+    static unsigned int pingpongFBO[2];
+    static unsigned int pingpongBuffer[2];
+    static unsigned int quadVAO;
+    static unsigned int quadVBO;
+    static unsigned int screenFBO;
+    static unsigned int screenTexture;
+
+    static engine::resources::Shader* bloom, *blur, *bloom_final;
+    static engine::resources::Shader* negative, *grayscale, *deepfried, *none, *outline, *blackWhite, *heartShader;
```

Comment:
[REFACTOR] [REFACTOR] Koji je razlog za čuvanje pokazivača na sve šejdere ovde? Pošto se instance već nalaze u ResourcesController-u, da li je zaista neophodno da ih i ovde posebno čuvamo, ili bi bilo bolje koristiti ih direktno iz ResourcesController-a?